calico-cloud/operations/cluster-management.mdx

```diff
@@ -16,15 +16,16 @@ If you inadvertenty delete a managed cluster, rerun the `curl` command and the c
 
 ### Access your managed clusters
 
-Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager.
+Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager UI.
 
 ![cluster-menu](/img/calico-cloud/cluster-menu.png)
 
-Use this menu to move between your managed clusters. You can create as many managed clusters as your subscriptions allows.
+Use this menu to move between your clusters. You can connect clusters as your subscriptions allows.
 
 ### Managed cluster views
 
-Each time you change to a different cluster, the entire Manager "view" is set for the cluster you've selected. For example, each managed cluster has unique indexes for Elasticsearch clusters, a separate instance of Kibana, and the dashboard reflects only the selected managed cluster.
+When you connect to a different cluster, the entire Manager UI view changes to reflect the selected cluster.
```

Comment:
Hmm, ambiguous language. As multiple clusters can be connected...
I perhaps reword this to:
'when you selected a cluster from the cluster dropdown, the entire Manager UI view changes to reflect the selected cluster.'
or
'when a diferent cluster is selected from the cluster dropdown, the entire Manager UI view changes to reflect the selected cluster.'

Reply:
Agreed. Changing to "When you select a different cluster from the cluster menu ..."